platform/security/core/security-core-impl/src/main/java/ddf/security/config/impl/ConfigurationSecurityLogger.java

```diff
@@ -56,6 +56,8 @@ public void configurationEvent(ConfigurationEvent event) {
         // there was no subject change was caused by an update to the config file on the filesystem
         SecurityLogger.auditWarn("Configuration {} via filesystem: {}", type, updatedConfiguration);
       }
+    } catch (VirtualMachineError vme) {
+      throw vme;
     } catch (Throwable e) {
       LOGGER.error("Error auditing config update for {}", event.getPid(), e);
```

Comment:
✏️  I know it is outside the scope of this PR, but the error log will not be logging the exception here. Replace with: ```LOGGER.error("Error auditing config update for " + event.getPid(), e);```

Reply:
Referencing Lombardi's comment on the ticket, should we try to be more specific and catch OutOfMemory, StackOverflow, and Thread Death as well?

Reply:
After discussing this with @lessarderic, @pklinef, and @figliold we concluded in only catching VirtualMachineError and throwing these back without doing anything or swallowing them.

Reply:
Ah ok, I didn't realize this was the **only** error that needed to be caught.